scalding-serialization-macros/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala

```diff
@@ -238,40 +248,49 @@ class MacroOrderingProperties extends FunSuite with PropertyChecks with ShouldMa
   test("Test out jl.Byte") {
     implicit val a = arbMap { b: Byte => java.lang.Byte.valueOf(b) }
     check[java.lang.Byte]
+    checkCollisions[java.lang.Byte]
```

Comment:
there are only 256 bytes, so taking 100 of them, the chance of collision seems pretty high. Seems like we should use the formula, pass the cardinality of T into checkCollisions, and then have it pick the k (100 in your example).